crates/cext/src/circuit.rs

```diff
@@ -1157,21 +1157,23 @@ pub unsafe extern "C" fn qk_circuit_delay(
 /// Convert a given circuit to a DAG.
 ///
 /// @param circuit A pointer to the circuit from which to create the DAG.
-/// @param copy_operations Performs a deep copy of operations if set to true
 /// @param qubit_order The order of the qubits in the output DAG. Defaults to the same order
 ///     as the input circuit if NULL. If not NULL, it must be of length ``qk_circuit_num_qubits()``.
 /// @param clbit_order The order of the clbits in the output DAG. Defaults to the same order
 ///     as the input circuit if NULL. If not NULL, it must be of length ``qk_circuit_num_clbits()``.
 ///
-/// @return A pointer to the new DAG.
+/// @return A pointer to the new DAG. The pointer needs to be freed with ``qk_dag_free``.
```

Comment:
I think usually we put the comment about freeing under the summary line, just before the params.